index.html

```diff
@@ -152,6 +152,18 @@ <h5>
     </ul>
     <hr>
   </section>
+<section class="form_section">
+  <h2>Cantact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+    coding. I’d love to help with it</p>
+    <form class="form_submit" action="https://formspree.io/f/xyyawdao" method="post">
+    <input class="input_element" name="name" placeholder="Full name" type="text" value="Yeremias" required maxlength="30">
+    <input class="input_element" type="email" name="email" placeholder="Email" required value="natajayanj14@gmail.com">
+    <textarea class="text_elemnt" name="message" placeholder="Write your message" id="" cols="30" rows="10" required maxlength="500"></textarea>
+    <button type="submit" class="btn_details submit_btn">Get in touch</button>
+    </form>
+    <hr class="bottom_line">
+</section>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure to stick as much as possible to the given Figma design. At the moment, the text for this section is different than the one in the template. Please consider fixing it! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/220450702-2fca120b-0b30-44be-9374-3f2024710f1b.png) | ![Screenshot from 2023-02-21 15-22-40](https://user-images.githubusercontent.com/77069011/220450632-67c3fad2-e408-41c1-b749-897aa0a49ad1.png) |

- [ ] Also, please add some padding to your inputs so the text doesn't look like is touching the edges! 🤓 🔧 

- [ ] **[OPTIONAL]:** Is okay if you delete the "value" property of your inputs, they are present in the design mainly to show the color of the font and the spacing of the input. In reality, having these arbitrary "default" values doesn't provide a great user experience! 🤓 🔧